docs/extendedjob.md

```diff
@@ -12,43 +12,43 @@
          2. [Reconciliation](#reconciliation-in-job-controller)
          3. [Highlights](#highlights-in-job-controller)
    3. [Relationship with the BDPL component](#relationship-with-the-bdpl-component)
-   4. [ExtendedJob Examples](#extendedjob-examples)
+   4. [QuarksJob Examples](#quarksjob-examples)
 
 ## Description
 
-An `ExtendedJob` allows the developer to run jobs when something interesting happens. It also allows the developer to store the output to a file /mnt/quarks/output.json which is transformed into a `Secret` later.
-The job started by an `ExtendedJob` is deleted automatically after it succeeds.
+An `QuarksJob` allows the developer to run jobs when something interesting happens. It also allows the developer to store the output to a file /mnt/quarks/output.json which is transformed into a `Secret` later.
+The job started by an `QuarksJob` is deleted automatically after it succeeds.
 
-There are two different kinds of `ExtendedJob`:
+There are two different kinds of `QuarksJob`:
 
 - **one-offs**: automatically runs once after it's created
 - **errands**: needs to be run manually by a user
 
-## ExtendedJob Component
+## QuarksJob Component
 
-The **ExtendedJob** component is a categorization of a set of controllers, under the same group. Inside the **ExtendedJob** component we have a set of 2 controllers together with one separate reconciliation loop per controller.
+The **QuarksJob** component is a categorization of a set of controllers, under the same group. Inside the **QuarksJob** component we have a set of 2 controllers together with one separate reconciliation loop per controller.
 
-Figure 1, illustrates the **ExtendedJob** component diagram and the set of controllers it uses.
+Figure 1, illustrates the **QuarksJob** component diagram and the set of controllers it uses.
 
-![flow-extendeds-job](quarks_ejobcomponent_flow.png)
-*Fig. 1: The ExtendedJob component*
+![flow-quarks-job](quarks_qjobcomponent_flow.png)
+*Fig. 1: The QuarksJob component*
 
 ### **_Errand Controller_**
 
-![errand-controller-flow](quarks_ejoberrandcontroller_flow.png)
+![errand-controller-flow](quarks_qjoberrandcontroller_flow.png)
```

Comment:
does `quarks_qjoberrandcontroller_flow.png` exists? If not, I will avoid this change

Reply:
I open a story to handle this https://www.pivotaltracker.com/story/show/169643007
Because we also need to update elements on these graphs.